releases/client/2.0/settings

```diff
@@ -0,0 +1,3 @@
+FULLVERSION="2.0-rc1"
+FOREMAN_VERSION="2.0"
+OSES="rhel5 rhel6 rhel7 el8 sles11 sles12 fedora29"
```

Comment:
We should prioritize updating to Fedora 30/31 in nightly since F29 is EOL. It feels weird to create a new release on an EOL distro.

Reply:
agreed, but not the scope of this pr ;)